package.json

```diff
@@ -1172,7 +1172,7 @@
                     "command": "jupyter.switchKernel",
                     "title": "%DataScience.selectKernel%",
                     "category": "Jupyter",
-                    "when": "jupyter.isnativeactive"
+                    "when": "jupyter.isnativeactive && !notebookEditorFocused"
```

Comment:
If this is the webview based command, then why does it need isnativeactive?

Reply:
Ohh, wait, is the name just off? isnativeactive covers any old / custom / native notebooks? If so that's kinda funky naming.